src/jobs/deploy_to_selfhosted.yml

```diff
@@ -3,87 +3,60 @@ description: "A job to deploy latest dev components to self-hosted"
 executor: do
 
 parameters:
-  release:
+  chart_name:
     description: The helm release name to be installed.
     type: string
-    default: "codacy-dev"
-  namespace:
-    description: The target namespace for the release.
+    default: "default-release"
+  environment:
+    description: The target environment for the release [dev / sandbox / release].
     type: string
-    default: "codacy-dev"
+    default: "dev"
+  helm_install_extra_flags:
+    description: Extra parameters to be passed to the helm install command.
+    type: string
+    default: ""
 
 steps:
-  - deploy:
+  - run:
       name: Set cluster to digital ocean
       command: |
         doctl auth init -t $DO_TOKEN &>/dev/null
         doctl kubernetes cluster kubeconfig save "$DOKS_CLUSTER_NAME" --set-current-context
-  - deploy:
-      name: Poll for installation availability
+  - run:
+      name: Add unstable repository
       command: |
-        helm poll -r << parameters.release >> -n << parameters.namespace >>
+        helm repo add codacy-unstable https://charts.codacy.com/unstable
+        helm repo update
   - deploy:
-      name: Trigger workflow to deploy dev components
+      name: Fetch values files
       command: |
-        function trigger_remote_workflow {
-            response=$(curl -XPOST -s -H 'Accept: application/json' -H 'x-attribution-login: string' -H 'x-attribution-actor-id: string' -H 'Content-type: application/json' -d '{"branch": "master"}' 'https://circleci.com/api/v2/project/gh/codacy/chart/pipeline' -H "Circle-Token: $CIRCLE_CI_GITHUB_TOKEN")
-            echo $response
-        }
-
-        function get_workflow_from_pipeline {
-            response=$(curl -s -H 'Accept: application/json' -H 'Content-type: application/json' "https://circleci.com/api/v2/pipeline/$1/workflow" -H "Circle-Token: $CIRCLE_CI_GITHUB_TOKEN")
-            echo $response
-        }
-
-        function get_workflow {
-            response=$(curl -s -H 'Accept: application/json' -H 'Content-type: application/json' "https://circleci.com/api/v2/workflow/$1" -H "Circle-Token: $CIRCLE_CI_GITHUB_TOKEN")
-            echo $response
-        }
-
-        function get_workflow_status {
-            response=$(get_workflow $1 | jq -r '.status')
-            echo $response
-        }
-
-        function get_remote_workflow_url {
-            workflow=$(get_workflow $1)
-            id=$(echo $workflow | jq -r '.id')
-            number=$(echo $workflow | jq -r '.pipeline_number')
-            echo "https://app.circleci.com/pipelines/github/codacy/chart/$number/workflows/$id"
-        }
+        curl -s -o values-production.yaml https://raw.githubusercontent.com/codacy/chart/master/codacy/values-production.yaml
+        curl -s -o values.yaml https://raw.githubusercontent.com/codacy/chart/master/codacy/values.yaml
```

Comment:
why do we need this values.yaml file?

Reply:
Without that file, portal could not connect to rabbitmq. The hostname for rabbit wouldn't get set.